guide/payments/request.md

```diff
@@ -116,52 +108,126 @@ A piece of meta data unique to invoices is an expiration time. This is how long
    image = "/assets/images/guide/payments/request/single-use-payment-request.png"
    retina = "/assets/images/guide/payments/request/single-use-payment-request@2x.png"
    layout = "shadow"
-   caption = "A newly generated single-use Lightning payment request."
+   caption = "A newly generated single-use payment request."
    alt-text = "Screen showing single-use payment request."
    width = 250
    height = 541
 %}
 
 {% include picture.html
-   image = "/assets/images/guide/payments/request/single-use-payment-request-settings.png"
-   retina = "/assets/images/guide/payments/request/single-use-payment-request-settings@2x.png"
+   image = "/assets/images/guide/payments/request/long-expiry.png"
+   retina = "/assets/images/guide/payments/request/long-expiry@2x.png"
    layout = "shadow"
-   caption = "Let users configure the settings of single-use Lightning payment requests."
-   alt-text = "Screen showing single-use payment request settings."
+   caption = "Zero value payment requests should be easily generated."
+   alt-text = "Screen showing invoice with long expiry."
    width = 250
    height = 541
 %}
 
+</div>
+
+### Invoice expirations
```

Comment:
This section is hard to read because it keeps referring to the volatility issue so often. How about creating two sub-sections. One for a regular bitcoin payment, the other for payments where the recipient wants to receive a specific amount in fiat.

It's also getting pretty long, we could easily split this into a new sub-page